Фикс влажной тряпочки

Miracler112 opened the pull request:
Теперь Агреволь может играть за бармена и наслаждаться протиранием стаканов в руках. А другие бармены получают еще больше простора для РП отыгрыша 

Ишью -  #390

🆑 

- bugfix: Влажной тряпочкой можно протереть стакан в руках

Commit:
Фикс damp rag

- Теперь им можно вытирать стаканы даже держа в руках
- Старые теги /red заменены на актуальный <span class>

LudwigVonChesterfield commented on code/modules/detectivework/footprints_and_rag.dm:
```diff
@@ -39,11 +39,11 @@
 	if(!proximity) return
 	if(user.is_busy()) return
-	if(user.client && (A in user.client.screen))
+	if(user.client && !istype(A, /obj/item/weapon/reagent_containers/food/drinks/drinkingglass) && (A in user.client.screen))
```
Мне в принципе интересны эти костыли с клиентом, и скрином.

Miracler112 replied:
Без понятия что это за костыли такие, но они были задолго до меня, видимо еще задолго до того, как тут Дред что-то шатал, а потому трогать здесь что-то не решился, ибо мало ли что-то слетит

Commit:
Удаляем лишнее условие

Костыль A in user.client.screen нужен, так как через него регистрируются все предметы, что есть у user, даже те, что находятся в поясе и других контейнерах внутри другого
